README.md

```diff
@@ -1 +1,120 @@
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+    <li><a href="https://www.rubyonrails.org/">Ruby on Rails</a></li>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+</ul>
+
+### Key Features <a name="key-features"></a>
+
+- Create a category as well as transaction inside the category
+- List all recipes by users
+- Displays the total price of all transactions in the category 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
+
+- [Live Demo Link](https://wealtho.onrender.com/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+### Prerequisites
+  * Make sure you have Ruby installed in your system. You can install it [here](https://www.ruby-lang.org/en/documentation/installation/)
+
+  * Get started with [Ruby on Rails](https://guides.rubyonrails.org/getting_started.html).
+
+  * Make sure you have [PostgreSQL](https://www.postgresql.org/) installed and running.
+
+### Setup
+
+  * Clone this repository by running `git clone https://github.com/Thinus01/Wealtho` in your command line.
+
+  * Run `bundle install` to install all the gems.
+### Run
+  *  Change/include `host, username, password` according to your database server.
+  *  Run `rails db:create` and `rails db:migrate`
+  *  Run `rails db:seed` to populate the database with some sample data.
+  *  Run `rails s` to start the server.
+  *  In your browser, go to `http://localhost:3000`.
+
+### Test
+
+  * Run `gem install rspec` to install Rspec,
+  * Run `rspec spec` to run all the test cases, and
+     
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Thinus Van De Venter**
+
+- GitHub: [@Thinus01](https://github.com/Thinus01)
+- Twitter: [@thinus_v_d_v](https://twitter.com/thinus_v_d_v)
+- LinkedIn: [Thinus Van De Venter](https://www.linkedin.com/in/thinus-van-de-venter-99aa26203)
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Optimized Database**
+- [ ] **Better UI Experience**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Thinus01/Wealtho/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, please consider giving it a star.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+- I would like to thank all code reviewers for making this project better.
```

Comment:
- [ ] Please give appropriate credit to [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) the [author of the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), as required by the [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/).